DFC.ServiceTaxonomy.GraphLookup/GraphSyncers/GraphLookupPartGraphSyncer.cs

```diff
@@ -47,6 +49,32 @@ public class GraphLookupPartGraphSyncer : IContentPartGraphSyncer
             return emptyResult;
         }
 
+        public Task<bool> VerifySyncComponent(ContentItem contentItem, INode node, ContentTypePartDefinition contentTypePartDefinition, IEnumerable<IRelationship> relationships, IEnumerable<INode> destNodes)
+        {
+            var nodes = (JArray)contentItem.Content.GraphLookupPart.Nodes;
+            var relationshipType = (string)contentTypePartDefinition.Settings["GraphLookupPartSettings"]!["RelationshipType"]!;
+
+            foreach (var n in nodes)
```

Comment:
Only a matter of style, but we shouldn't have 1 letter variable names (outside of lambdas).